<launch>
  <arg name="ns" default="map_merger"/>

  <!-- Arguments -->
  <!--<arg name="first_tb3"  default="tb3_0"/>
  <arg name="second_tb3" default="tb3_1"/>
  <arg name="third_tb3"  default="tb3_2"/>-->

  <node pkg="multi_robot" type="map_merger" respawn="false" name="map_merger" output="screen">
    <param name="estimation_confidence" value="1.0"/>
  </node>

</launch>
